DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/Helpers/RevitCategoryInfo.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Autodesk.Revit.DB;
+
+namespace Speckle.Converters.RevitShared.Helpers;
+
+public class RevitCategoryInfo
```

Comment:
Feels like we ought to inject this at some point or it will make unit testing anything that uses this more complicated